### PR TITLE
feat : design author more details UI

### PR DESCRIPTION
**Pull request by cp-mansi-d**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [ ] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [ ] Make MR mark as ready before submitting it for review
- [ ] Add images, videos, or gifs of UI
- [ ] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook (chrome, firefox, safari)
- [ ] iPhone(safari)
- [x] Available android devices (chrome, firefox)
- [x] iPad(safari)
- [ ] Tablet (chrome, firefox)

**Description :**
- design author details section.
- add role and bio field in author table and display that data in author details section.
- display lorem instead of bio only in record video.
- resolve env variable issues.

https://github.com/canopas/canopas-blog/assets/115449373/c417d4ec-2c47-41e8-a81b-d4a3909987c4

### CODE REVIEW

**Review by cp-priyanka-g — APPROVED** [review]

File: utils.js [hunk 1]
```diff
@@ -53,6 +53,8 @@ function setPostFields(post, slug) {
   post.attributes.authorAltText = author
     ? author.username + " image"
     : "author";
+  post.attributes.authorBio = author?.bio || "Author bio..";
```
Comment (cp-sumi-k) [inline utils.js:56]: Let's leave it blank if the bio is not there. And see how UI will look like if bio is blank

File: components/authorDetails.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            </div>
+            <div className="!mt-7 text-[0.9rem] capitalize">
+              {postData.authorRole}
+            </div>
+          </div>
+        </div>
+        <hr className="bg-gray-600 w-full py-[0.5px] !mb-3 mt-0" />
+      </div>
+      <div className="hidden sm:block container mx-5">
+        <hr className="bg-gray-600 w-full py-[1px] !my-3" />
```
Comment (cp-jagruti-a) [inline components/authorDetails.js:33]: use default class for padding

File: components/authorDetails.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            </div>
+            <div className="text-[1rem] leading-snug tracking-wide">
+              {postData.authorBio}
+            </div>
+            <div className="text-[0.9rem] capitalize">
+              {postData.authorRole}
+            </div>
+          </div>
+        </div>
+        <hr className="bg-gray-600 w-full py-[1px] !my-3" />
```
Comment (cp-jagruti-a) [inline components/authorDetails.js:56]: same applied here

File: utils.js (hunk 1 above)
Comment (cp-mansi-d) [reply utils.js:56]: it look like 
image

Comment (cp-sumi-k) [reply utils.js:56]: You can remove  space between them in this case

**Review by cp-jagruti-a — APPROVED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**